packages/machine-extractor/src/transitions.ts

```diff
@@ -22,6 +22,7 @@ const TransitionObject = objectTypeWithKnownKeys({
   target: TransitionTarget,
   actions: MaybeArrayOfActions,
   cond: Cond,
+  description: StringLiteral,
```

Comment:
This probably should be:
```suggestion
  description: unionType([StringLiteral, TemplateLiteral]),
```

and we'd need to add a test for that TemplateLiteral case